src/frontend/containers.rs

```diff
@@ -350,13 +351,13 @@ impl<'a, T: 'a + Source> From<&'a mut T> for RadecoModule<'a, DefaultFnTy> {
                 // TODO: Still keep track of these functions.
                 continue;
             }
-            radeco_trace!("Locals of {:?}: {:?}", f.name, f.locals);
+            //radeco_trace!("Locals of {:?}: {:?}", f.name, f.locals);
             let offset = f.offset.expect("Invalid offset");
             let instructions = source.instructions_at(offset);
             let tx = tx.clone();
             let reg_info = reg_info.clone();
 
-            let handle = thread::spawn(move || {
+            let handle = thread::Builder::new().name(f.name.as_ref().unwrap().to_owned()).spawn(move || {
```

Comment:
may be split this line?

Reply:
This doesn't matter. This whole file is set to be deprecated and we shouldn't be using it at all.